flaskr/steno.py

```diff
@@ -26,7 +26,7 @@ def index():
 def splash():
     db = get_db()
     posts = db.execute(
-        'SELECT p.id, title, body, created, author_id, username'
+        'SELECT p.id, title, body, created, author_id, username, otp'
```

Comment:
Do not select `p.id` here, you are no longer using it in the template, so it's not needed at all.